.env.production.template

```
@@ -0,0 +1,181 @@
+FIREBASE_SERVICE_ACCOUNT_KEY='{"type":"service_account","project_id":"{{FIREBASE_PROJECT_ID}}","private_key":"{{FIREBASE_PRIVATE_KEY}}","client_email":"{{FIREBASE_CLIENT_EMAIL}}"}'
+
+# Payment providers
+ASAAS_ENV=production
+ASAAS_API_KEY_PROD={{ASAAS_PROD_KEY}}
+ASAAS_API_KEY_SANDBOX={{ASAAS_SANDBOX_KEY}}
+ASAAS_WEBHOOK_TOKEN={{ASAAS_WEBHOOK_TOKEN}}
+STRIPE_SECRET_KEY={{STRIPE_SECRET_KEY}}
+STRIPE_WEBHOOK_SECRET={{STRIPE_WEBHOOK_SECRET}}
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY={{STRIPE_PUBLISHABLE_KEY}}
+NEXT_PUBLIC_STRIPE_BILLING_PORTAL_URL=https://billing.stripe.com/p/login/{{PORTAL_CODE}}
+NEXT_PUBLIC_ASAAS_API_KEY={{ASAAS_PUBLIC_KEY}}
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_ASAAS_API_KEY key should go before the NEXT_PUBLIC_STRIPE_BILLING_PORTAL_URL key

(UnorderedKey)